packages/home-spa/src/components/op-copyright/README.md

```diff
@@ -0,0 +1,29 @@
+# Welcome to op-copyright 👋
+![Version](https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000)
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/1-Platform/one-platform#readme)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/one-platform/graphs/commit-activity)
+[![License: MIT](https://img.shields.io/github/license//op-copyright)](https://github.com/1-Platform/one-platform/blob/master/LICENSE)
+
+> Copyright component for One-Platform
+
+### 🏠 [Homepage](https://github.com/1-Platform/one-platform#readme)
+
+## Author
+
+👤 **Deepesh Nair**
+
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check [issues page](https://github.com/1-Platform/one-platform/issues). You can also take a look at the [contributing guide](https://github.com/1-Platform/one-platform/blob/master/CONTRIBUTING.md).
```

Comment:
@hybridx Why not simply use the component as an npm dependancy instead of including it within the src/ ?

Reply:
The copyright component would be removed in the future when we have the footer component available in our Web Component Library